test/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -121,6 +121,45 @@ status:
       share1: 1
    transportURLSecret: rabbitmq-transport-url-manila-manila-transport
 ---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: manila-cron
+spec:
+  jobTemplate:
+    metadata:
+      labels:
+        service: manila
+    spec:
+      completions: 1
+      parallelism: 1
+      template:
+        spec:
+          containers:
+          - args:
+            - -c
+            - /usr/bin/manila-manage --debug --config-dir /etc/manila/manila.conf.d
```

Comment:
good to have this verification ++

Reply:
I had this question from the original PR  regarding `--debug`; we're not setting it properly; i.e., if debug for dbSync is True, only "--debug" is set

https://github.com/openstack-k8s-operators/manila-operator/pull/142/files#diff-e37c895a02f32ad01467f37d9743af26a4744a0a77207b25bfdaed6574882778R43-R45

alongside, i wonder if we want to specify a "dbPurge" entry under https://github.com/openstack-k8s-operators/manila-operator/blob/91a921182ce70edf5b5858edf93f0e6a55bc09ed/api/v1beta1/common_types.go#L119-L124

it'd be useful to see the o/p of the pruning from time to time.. 
```
        -            - /usr/bin/manila-manage --debug --config-dir /etc/manila/manila.conf.d
        -              db purge 30
        +            - /usr/bin/manila-manage --config-dir /etc/manila/manila.conf.d db purge
        +              30
```

Reply:
Yes, makes sense to decouple the `DBSync` job from the cronjob. I'll do in my next PS (at the beginning I thought that having a single bool to debug all the jobs was fine).
It's better having a separated bool to test/debug the cronjob Pod when it's triggered